src/components/Chart/Chart.js

```diff
@@ -1,41 +1,152 @@
-import React from 'react';
-import './Chart.css';
-import { Bar } from 'react-chartjs-2';
+import React, { Component } from 'react';
+import { categories, data, max } from '../../config.js';
 
-function Chart() {
-  const state = {
-    labels: ['January', 'February', 'March',
-             'April', 'May'],
-    datasets: [
-      {
-        label: 'Rainfall',
-        backgroundColor: 'rgba(75,192,192,1)',
-        borderColor: 'rgba(0,0,0,1)',
-        borderWidth: 2,
-        data: [65, 59, 80, 81, 56]
-      }
-    ]
+
+export default class Chart extends Component {
+
+  state = {
```

Comment:
You could do this using a map() of data, something like 

data.map(column => {state[column.id] = {amount: column.initial_amount, ..etc...}});

Reply:
sorry, I should have said `setState`, not `state[x] = ...`

Reply:
@HannahSchafer I'm realizing this is more of a refactor than a change request,and I know you're busy with data stuff right now, so I'm going to approve, and perhaps add this refactor when I merge our code, if that's cool with you.